rotator/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/jrick/logrotate/rotator
```

Comment:
Since the whole project on its own is already rather small, does it really make sense to turn this into its own Gomodule? IMO just the root one should be enough.